.travis.yml

```diff
@@ -1,36 +1,116 @@
 language: scala
-
 sudo: false
-
-scala:
-  - "2.11.12"
-  - "2.12.5"
-
-jdk:
-  - oraclejdk8
+jdk: oraclejdk8
 
 services:
   - docker
 
 before_install:
-  - docker-compose up -d
+  # make comparing to origin/master work
+  - git remote set-branches --add origin master && git fetch
```

Comment:
Hm are you sure this works? https://travis-ci.org/akka/akka-grpc/jobs/371532829 uploaded to `akka-grpc-adhoc` instead of `akka-grpc-master` @2m

Reply:
Thanks for checking. I probably missed the `git branch -f "$TRAVIS_BRANCH" && git checkout "$TRAVIS_BRANCH"` which needs to run before whitesource.